tests/GenerateScriptRegressionTests/GenerateScriptRegressionTestsFixture.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using Xunit.Abstractions;
+
+namespace GenerateScriptRegressionTests;
+
+public class GenerateScriptRegressionTestsFixture
```

Comment:
Utilizing a TestFixture for static Environmental information feels like overkill.  There is no need for the fixture overhead for static information.

Reply:
TestFixture was used for cleanup environment because in the first design, we just use git command to check file changes. But now since we're using sandbox design, we could get rid of this TestFixture. Will update.